ui/components/multichain/create-btc-account/create-btc-account.test.tsx

```diff
@@ -79,26 +68,39 @@ describe('CreateBtcAccount', () => {
 
   it('fires onActionComplete when clicked', async () => {
```

Comment:
To not confuse with the "Add account" click test:

```suggestion
  it('fires onActionComplete when "Cancel" has been clicked', async () => {
```